QuanLyNhanVien/DAL/KetNoi.cs

```diff
@@ -77,6 +77,7 @@ public bool ExcuteNonQuery(string query, object[] parameter = null)
                             i++;
                         }
                     }
+                    Console.WriteLine("Chỗ này thêm mới cốnle ưrtie linecho đẹp hơn về ặt coding");
```

Comment:
why did you use vnese?

Reply:
Because of the QA require me use vnese here.

Reply:
@HauVuTri  Are you ok with iut?